index.md

```diff
@@ -40,6 +40,25 @@ is contained within parenthesis, e.g., `feat(parser): add ability to parse array
 Commit _types_ other than `fix:` and `feat:` are allowed, for example [the Angular convention](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit) recommends `docs:`, `style:`, `refactor:`, `perf:`, `test:`, `chore:`, but these tags are
 not mandated by the conventional commits specification.
 
+## Examples
+
+### Commit message with no body
+```
+fix: Correct spelling of CHANGLOG
+```
+
+### Commit message with body and breaking change
+```
+feat: allow provided config object to extend other configs
+
+BREAKING CHANGE: `extends` key in config file is now used for extending other config files
+```
+
+### Commit message with scope
```

Comment:
I think this is the most commonly used message so I'd reorder this to top :)

Reply:
I'll fix this.